flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -0,0 +1,92 @@
+DROP TABLE IF EXISTS public."user_role_relation";
+DROP TABLE IF EXISTS public."user_organization_relation";
+DROP TABLE IF EXISTS public."sso_role";
+DROP TABLE IF EXISTS public."sso_user_profile";
+DROP TABLE IF EXISTS public."sso_organization";
+DROP TABLE IF EXISTS public."sso_user";
```

Comment:
Không nên xóa bảng vì có thể gây mất dữ liệu. Chỉ cần câu lệnh `CREATE TABLE IF NOT EXISTS` thì sẽ không bị lỗi khi bảng đã tồn tại.
```suggestion
```